content/android/getting-started.md

```diff
@@ -1,116 +1,111 @@
 Skygear aims to provide a complete and open-source backend solution for your mobile applications.
 
-By creating an App for each of your mobile applications on Skygear, each client App can be connected to the Skygear Server with an API key.
+By creating an app for each of your mobile applications on Skygear, each client App can be connected to the Skygear Server with an API key.
 
 This guide will show you how to integrate Skygear into an existing iOS project.
 
 <a name="signing-up"></a>
 ## Signing up for Skygear Hosting
 
-Sign up an account at [Skygear Portal](http://portal.skygear.io/).
-
-You will need the server endpoint and API Key to set up your app.
+To start using the Skygear Android SDK, you need to register your account and
+application at the Skygear [Developer Portal](https://portal.skygear.io)
+website. After you registered, go to the **INFO** tab and copy down your
+`Server EndPoint` and `API Key`.
 
+[![Screenshot: where to look for Server EndPoint and API Key](/assets/common/portal-endpoint-apikey.png)](/assets/common/portal-endpoint-apikey.png)
 
 <a name="including"></a>
-## Including the SDK in your project
-
-1. Make sure jcenter repository is included in `build.gradle` of your project
-
-  ```gradle
-  allprojects {
-      repositories {
-          jcenter()
-      }
-  }
-  ```
-
-2. Add skygear as dependency in `build.gradle` of your application
-
-  ```gradle
-  dependencies {
-      // other dependencies
-      compile 'io.skygear:skygear:+'
-  }
-  ```
+## Including SDK in existing project
 
-<a name="setting-up"></a>
-## Setting up Skygear
+### Step 1: Installing SDK
 
-### Option 1: Using `SkygearApplication` as Custom Application
+Make sure jcenter repository is included in `build.gradle` of your project
 
-If you does not have a custom application class, you can set up skygear as following:
+```gradle
+allprojects {
+    repositories {
+        jcenter()
+    }
+}
+```
 
-1. Create custom class extends `SkygearApplication`
+Add skygear as dependency in `build.gradle` of your application
 
-  ```java
-  import io.skygear.skygear.SkygearApplication;
+```gradle
+dependencies {
+    // other dependencies
+    compile 'io.skygear:skygear:+'
+}
+```
 
-  public class MyApplication extends SkygearApplication {
-      // ...
-  }
-  ```
+### Step 2: Configuring container
 
-2. Update `AndroidManifest.xml`
+After you have installed the SDK, you must configure
+your skygear container with the `Server EndPoint` and `API Key` you get on
+Skygear Developer Portal **BEFORE** you make any API calls.
 
-  - add `android:name` attribute to `application` tag
+#### Option 1: Using `SkygearApplication` as custom application
```

Comment:
I think Option 1 / 2 are served as different approaches. 

Option 1 is like a minimal setup of the SDK.
Option 2 is more suitable for integrating the SDK to existing application.